src/main/java/com/mcharopkin/server/MainServlet.java

```diff
@@ -30,6 +30,11 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
 		resp.addHeader("Access-Control-Allow-Origin", "192.168.56.101:5080");
 		resp.addHeader("Access-Control-Allow_Methods", "*");
 		resp.setCharacterEncoding("UTF-8");
+		try {
+			System.out.println("hui");
+		} catch (Exception ex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.foods.devops.mnscorp.net/coding_rules#rule_key=squid%3AS1166)